src/test/java/org/reactome/addlinks/test/TestEnsemblBioMartRetriever.java

```diff
@@ -0,0 +1,97 @@
+package org.reactome.addlinks.test;
+
+import org.junit.Test;
+import org.reactome.addlinks.dataretrieval.ensembl.EnsemblBioMartRetriever;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class TestEnsemblBioMartRetriever {
+
+    private static final String testDirectory = "/tmp/test_ensembl_biomart_mapping_service/";
+    private static final String expectedErrorMessage = "java.lang.Exception: BioMart query failed with message: Query ERROR";
+
+    //TODO Move these to IT?
```

Comment:
Yeah, I think moving any tests to be integration tests that need to access an outside resource like a web service, database, etc. would be good.

Reply:
Done